nohost/src/rewriter.js

```diff
@@ -97,7 +97,7 @@ define(function (require, exports, module) {
 
         // Turn this html into a DOM, process it
         var parser = new DOMParser();
-        this.doc = parser.parseFromString(html, 'text/html');
+        this.doc = parser.parseFromString(html.body, 'text/html');
```

Comment:
I don't understand this change.  Can you explain it?  `html` should be a raw string of HTML content, not a DOM node.